lib/components/molecules/ArticleThumbnail/ArticleThumbnail.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+/**
+ *
+ * ArticleThumbnail
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './ArticleThumbnail.style';
+import type { Props } from './types';
+import Image from '../../atoms/Image/Image';
+import Heading from '../../atoms/Heading/Heading';
+import Anchor from '../../atoms/Anchor';
+import Para from '../../atoms/Para';
+
+const getAnchoredChild = (url, children, target) => (
```

Comment:
What if someone wants to define/pass more props on the link like a class? We are not using full capabilities of link in this style.

Reply:
Done